common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -615,7 +622,11 @@ def handle_problem_html_error(self, err):
                 html += self.lcp.get_html()
             except Exception:
                 # Couldn't do it. Give up.
-                log.exception("Unable to generate html from LoncapaProblem")
+                log.exception(
+                    u"Unable to generate html from LoncapaProblem: !r, !r",
```

Comment:
Can we add `ProblemGetHtmlError` in this log as well so we can get all the logs quickly?

Reply:
Yes.

Reply:
PR updated.

Reply:
Weren't we supposed to log `html` and `lcp.get_html()` to get the content information? We should log them only for the course where we are facing. This will give us all the missing info in the first go and we wouldn't have to rely on the follow-up logs.

Reply:
I had this idea, but the content of the problem may vary and can expand to 10s of lines, So we have the problem location, which I am hopeful will lead us to the exact problem content.

Reply:
Ok, for now. But provided that the underlying issue is happening for one learner only, I doubt that only location information will be helpful.